tests/providers/aws/services/s3/s3_access_point_public_access_block/s3_access_point_public_access_block_test.py

```diff
@@ -316,3 +316,78 @@ def test_access_points_without_one_public_access_block(self):
+        arn_ap = f"arn:aws:s3:ap-southeast-2:{AWS_ACCOUNT_NUMBER}:accesspoint/{ap_name_ap}"
+
+        with mock.patch(
+            "prowler.providers.common.provider.Provider.get_global_provider",
+            return_value=aws_provider,
+        ), mock.patch(
+            "prowler.providers.aws.services.s3.s3_access_point_public_access_block.s3_access_point_public_access_block.s3control_client",
+            new=S3Control(aws_provider),
+        ):
+            from prowler.providers.aws.services.s3.s3_access_point_public_access_block.s3_access_point_public_access_block import (
+                s3_access_point_public_access_block,
+            )
+
+            s3control_client = mock.MagicMock()
+            s3control_client.access_points = {
+                arn_us: AccessPoint(
+                    account_id=AWS_ACCOUNT_NUMBER,
+                    name=ap_name_us,
+                    bucket="bucket-us",
+                    region="us-east-1",
+                    public_access_block=PublicAccessBlock(
+                        block_public_acls=True,
+                        ignore_public_acls=True,
+                        block_public_policy=True,
+                        restrict_public_buckets=True,
+                    ),
+                ),
+                arn_eu: AccessPoint(
+                    account_id=AWS_ACCOUNT_NUMBER,
+                    name=ap_name_eu,
+                    bucket="bucket-eu",
+                    region="eu-west-1",
+                    public_access_block=PublicAccessBlock(
+                        block_public_acls=False,
+                        ignore_public_acls=False,
+                        block_public_policy=False,
+                        restrict_public_buckets=False,
+                    ),
+                ),
+                arn_ap: AccessPoint(
+                    account_id=AWS_ACCOUNT_NUMBER,
+                    name=ap_name_ap,
+                    bucket="bucket-ap",
+                    region="ap-southeast-2",
+                    public_access_block=PublicAccessBlock(
+                        block_public_acls=True,
+                        ignore_public_acls=False,
+                        block_public_policy=True,
+                        restrict_public_buckets=True,
+                    ),
+                ),
+            }
+
+            with patch(
+                "prowler.providers.aws.services.s3.s3_access_point_public_access_block.s3_access_point_public_access_block.s3control_client",
+                s3control_client,
+            ):
+                check = s3_access_point_public_access_block()
+                result = check.execute()
+
+                assert len(result) == 3
+                assert result[0].status == "PASS"
+                assert result[1].status == "FAIL"
+                assert result[2].status == "FAIL"
+                assert result[2].region == "ap-southeast-2"
```

Comment:
Could you add more assertions? (`status_extended`, `resource_id`,  `resource_arn`...) take a look at `test_access_points_without_public_access_block` and try to add all those asserts 🙏🏼